content/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app.md

```diff
@@ -84,15 +84,16 @@ puts jwt
 
 {% note %}
 
-**Note:** You must run `pip install jwt` to install the `jwt` package in order to use this script.
+**Note:** You must run `pip install PyJWT` to install the `jwt` package in order to use this script.
```

Comment:
```suggestion
**Note:** You must run `pip install PyJWT` to install the `PyJWT` package in order to use this script.
```

Reply:
@m-balghonaim Thank you for this suggestion! 💛 

@altaurog If you are able to commit the suggestion above, we can go ahead and get this merged ✨ thank you!

Reply:
haha, good catch.  Done with thanks.